airflow/contrib/operators/spark_sql_operator.py

```diff
@@ -87,13 +86,8 @@ def execute(self, context):
                             master=self.master,
                             yarn_queue=self.yarn_queue
                             )
-        process = hook.run_query()
+        self.hook.run_query()
 
-        # using two iterators here to support 'real-time' logging 
-        for line in iter(process.stdout.readline, b''):
```

Comment:
No, I moved it to the hook, as that's where the subprocess object now is (instead of in the operator), similar to other hooks/operators.